lib/facter/env_temp_variable.rb

```diff
@@ -0,0 +1,3 @@
+Facter.add(:env_temp_variable) do
+  setcode { ENV['TEMP'] }
```

Comment:
Looks like you are not able to use that - http://ruby-doc.org/stdlib-2.1.6/libdoc/tmpdir/rdoc/Dir.html#method-c-tmpdir (resolves to ENV['TEMP'] on Windows). 

https://github.com/ruby/ruby/blob/v2_1_6/lib/tmpdir.rb#L20-L37